extensions/sql-database-projects/package.json

```diff
@@ -24,11 +26,11 @@
     "type": "git",
     "url": "https://github.com/Microsoft/azuredatastudio.git"
   },
-  "extensionDependencies": [
+	"extensionDependencies": [
```

Comment:
Are we mixing tabs + spaces in this file?

Reply:
somehow this change can only be seen here, looks fine in the editor. But shift+alt+f corrected the formatting, Thanks for noticing it.